tests/components/switch/common.py

```diff
@@ -42,8 +42,8 @@ async def async_turn_off(hass, entity_id=ENTITY_MATCH_ALL):
     await hass.services.async_call(DOMAIN, SERVICE_TURN_OFF, data, blocking=True)
 
 
-def get_mock_switch_entities() -> list[MockToggleEntity]:
-    """Return a list of mock switch entities."""
+def get_mock_toggle_entities() -> list[MockToggleEntity]:
```

Comment:
In the future we should change this to use actual SwitchEntity instead, since that's what the switch platform uses.

Reply:
Yes, definitely, I'll have a look when I've finished removing `hass.components`.